tests/test_statepoint_restart/test_statepoint_restart.py

```diff
@@ -52,11 +59,13 @@ def _run_openmc_restart(self):
                                                  mpi_exec=self._opts.mpi_exec)
 
         else:
-            returncode = executor.run_simulation(openmc_exec=self._opts.exe)
+            returncode = executor.run_simulation(openmc_exec=self._opts.exe,
+                                                 restart_file=statepoint)
```

Comment:
The changes above are actually more important.  It looks like we don't actually restart if you pass a list rather than a string